release_scripts/signassert.py

```diff
@@ -0,0 +1,63 @@
+################################################################################
+#                                                                              #
+# Copyright (C) 2011-2015, Armory Technologies, Inc.                           #
+# Distributed under the GNU Affero General Public License (AGPL v3)            #
+# See LICENSE or http://www.gnu.org/licenses/agpl.html                         #
+#                                                                              #
+################################################################################
+
+import getpass
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
+
+from armoryengine.ALL import *
```

Comment:
I saw your comment elsewhere about passing args in. I believe that, if you import armoryengine/armoryutils.py, you get --argparse as part of it. You may have to narrow down the import and import only what you need. (That's a good practice anyway.) The alternative - adding flags to Armory in general - probably isn't a good idea.